organisms/organisation-unit-tree/new-features/partial-selectability.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Disable selection for some org units in a tree
```

Comment:
I think this is too specific and can be simplified:
```
Feature: Org units can be disabled

  Sometimes the entire tree must be displayed but only certain units are available for selection

  Scenario: Some org units are disabled
    Given some org units are disabled
    Then the user can't select these org units
```

We should also think about what happens to these org units when a group/level selection is made while some units are disabled (where does the disabled state come from? Does the BE know about this (validation)?).
Generally I think that group/level selection should be handled by the back end (@larshelge, i. e. we could send sth like: `['SELECT_DESCENDANTS_BY_TYPE', 'org_unit_id'])` and the back end should know which org units are select-able, so that the org unit tree component doesn't have to worry about it

Reply:
Much simpler, thanks. Updated.

I agree that BE should handle disabled units on group/level selection. Disabled state could, for example, come from that org.unit not being valid with a previous selection, perhaps a tracker program. There are cases where it's still relevant to display, but not allow selection. I would say the most we want to do on the FE is possibly display a 'Your group selection may include disabled org units'. I'm not sure if it would be necessary.

Reply:
Sounds good to me